homeassistant/components/person/__init__.py

```diff
@@ -446,6 +447,8 @@ def extra_state_attributes(self):
             data[ATTR_SOURCE] = self._source
         if (user_id := self._config.get(CONF_USER_ID)) is not None:
             data[ATTR_USER_ID] = user_id
+        if self.device_trackers is not None:
+            data[ATTR_DEVICE_TRACKERS] = self.device_trackers
```

Comment:
We should have the attribute consistently available, otherwise, you'll end up checking if the attribute is present or not in templates.

```suggestion
        data[ATTR_DEVICE_TRACKERS] = self.device_trackers
```